tests/06_from_raw_to_list_test.py

```diff
@@ -0,0 +1,54 @@
+"""Test from_raw_to_list.
+
+This module tests the correctness and exceptions of ReadCourseData/from_raw_to_list()
+"""
+
+
+import sys
+sys.path.append('./src')
+from ReadCourseData import from_raw_to_list
+from InsertData import check_file_open
+import pytest
+
+def test_from_raw_to_list_correctness():
+    """Test if from_raw_to_list() returns the right content."""
+    course_raw = check_file_open('tests/test.json')
+    quarter_name = 'Test Data'
+    course_list, department_list = from_raw_to_list(course_raw, quarter_name)
+    assert len(course_list) == 6
+    assert len(department_list) == 2
+    assert course_list[5].UID == department_list[1].courses[2].UID == '35528'
+    assert course_list[5].crn == department_list[1].courses[2].crn == '35528'
+    assert course_list[5].course_num == department_list[1].courses[2].course_num == 'D064Y'
+    assert course_list[5].section_num == department_list[1].courses[2].section_num == '56'
+    assert course_list[5].campus == department_list[1].courses[2].campus == 'DA'
+    assert course_list[5].num_credit == department_list[1].courses[2].num_credit == 3.000
+    assert course_list[5].course_title == department_list[1].courses[2].course_title == 'ADMJ INTRNSHIP'
+    assert course_list[5].startTime == department_list[1].courses[2].startTime == '12:30 pm'
+    assert course_list[5].endTime == department_list[1].courses[2].endTime == '03:20 pm'
+    assert course_list[5].cap == department_list[1].courses[2].cap == 20
+    assert course_list[5].wl_cap == department_list[1].courses[2].wl_cap == 0
+    assert course_list[5].instructor_name == department_list[1].courses[2].instructor_name == 'Terry,R,Ellis'
+    assert course_list[5].startDate == department_list[1].courses[2].startDate == '01/08'
+    assert course_list[5].endDate == department_list[1].courses[2].endDate == '03/30'
+    assert course_list[5].location == department_list[1].courses[2].location == 'DA L13C'
+    assert course_list[1].location == department_list[0].courses[1].location == 'DA ONLINE'
+    assert course_list[5].days == department_list[1].courses[2].days == 'MWF'
+    assert course_list[1].days == department_list[0].courses[1].days == 'ONLINE'
+    assert course_list[5].attribute == department_list[1].courses[2].attribute == ''
```

Comment:
Should we use a list of test inputs and a list of target results and loop through both?

Reply:
Yes